CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [2.3.2] - 2020-04-18
+### Added
+- Japanese bandplan information to display with the `?plan jp` command
+### Fixed
+- Details to the Nederlands bandplan command to accurately represent VERNON (Nederlands ARRL equivalent organisation).
```

Comment:
Also, `Nederlands` -> `Netherlands` 2x in this line

Reply:
Fixed below.